e2e_tests/src/e2e_indexer/e2elive.py

```diff
@@ -151,17 +108,7 @@ def main():
     try:
         logger.info("reached expected round={}".format(lastblock))
         if not args.read_only:
-            xrun(
-                [
-                    "validate-accounting",
-                    "--verbose",
-                    "--algod",
-                    algoddir,
-                    "--indexer",
-                    indexerurl,
-                ],
-                timeout=20,
-            )
+            nxrun(validate_accounting, verbose=True, algod=algoddir, indexer=indexerurl)
```

Comment:
nxrun has a idempotentcy requirement for func over xrun if run multiple times, but I assume validate_accounting meets that requirement

Reply:
Not sure I see the idempotency requirement. Does `xrun` have this? If so, I'll probably want to rename `nxrun` to `run_with_exit_codes` or something like that.

Reply:
After convo I understand the question better. Yes, this new approach is introducing the possibility of side effects in the running python environment, while the former approach shielded the environment. In a production setting, this may be undesirable. But for the purposes of testing, I think it's okay.

Reply:
I'll add a comment to `def xrun()` alerting of this issue.

Reply:
cf. 69a51075b9601ccb0385f88f9d2db4d5668a09df